### DISCUSSION TITLE
Turborepo, next js, wagmi and rainbowkit webpack error

### DISCUSSION THREAD

**Discussion by IbrahimIjai** [wevm/viem#949] [General]
I am just trying to set up a project using turborepo and next js, but after getting my wagmi and rainbowkit setup and wrapped up around my provider, I keep on getting this error. Please how can I fix???? 

- error ../../node_modules/viem/dist/cjs/utils/abi/getAbiItem.js
Module parse failed: Unexpected character '' (1:0)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
(Source code omitted for this binary file)

**Comment by tmm** [comment]
Looks like a tooling issue on your end that's not related to wagmi/viem.
